Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2_test.py

```diff
@@ -1450,11 +1451,12 @@ def test_fetch_new_incidents(requests_mock):
     last_run = {'time': '2021-09-26T14:02:59.000000Z', 'id': '2'}
     fetch_time = '3 days'
     max_fetch = 2
+    filter_label_name = ''
```

Comment:
lets assert the url we got with these 2 tests with requestsmock

Reply:
They're being asserted, requests_mock.get won't retrieve the incidents if the call wasn't made to the exact URL.|
I'm even tried to change the URL in the requests_mock.get and saw that the unit-test failed

Reply:
if it wasnt added it would still succeed because
```
 requests_mock.get(BASE_URL + '/plugin/products/detect3/api/v1/alerts?'
                                 '&state=unresolved&sort=-createdAt&limit=500&offset=0'
```
is there without the suffix, no?

Reply:
There're two unit tests, one of a fetch_incidents with an empty filter_label_name that ensures that the 'labelName' query param is not being added to the URL suffix.

And the second one with a labelName that ensures the 'labelName' query param is being added to the URL suffix. at the right way.